wsbroadcastserver/wsbroadcastserver.go

```diff
@@ -250,6 +275,10 @@ func (s *WSBroadcastServer) StartWithHeader(ctx context.Context, header ws.Hands
 			return
 		}
 
+		compressionAccepted := false
+		if compress != nil {
+			_, compressionAccepted = compress.Accepted()
```

Comment:
Could we disconnect the client early here if we require compression but it wasn't accepted?

Reply:
sounds like a good idea  - I've added the early disconnection. I just wonder if we should notify the client somehow about the disconnection reason or is it acceptable for those clients to keep retrying? The server's `RequireCompresion` option could be hot reloaded to false and then it would make sense for the clients to keep trying, but it would also be nice to log the reason on the client side.

Reply:
recaping standup discussion:
* client backoff mechanism should mitigate possible issue
* the old clients that most probably would be the ones disconnected, want be able to properly handle disconnection reason